content/en/blog/_posts/2024-08-13-Kubernetes-v1-31-Release.md

```diff
@@ -0,0 +1,442 @@
+
+This work was done as a part of [KEP #2644](https://github.com/kubernetes/enhancements/issues/2644) and by [SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage).
+    
+### Bound service account token improvements
+
+The `ServiceAccountTokenNodeBinding` feature is getting promoted to beta in 1.31. 
+This feature allows requesting a token bound only to a node, not to a pod, which includes node information in claims in the token and validates the existence of the node when the token is used. 
+For more inforamtion, read the [bound service account tokens documentation](/docs/reference/access-authn-authz/service-accounts-admin/#bound-service-account-tokens).
+
+
+This work was done as part of [KEP #4193](https://github.com/kubernetes/enhancements/issues/4193) by [SIG Auth](https://github.com/kubernetes/community/tree/master/sig-auth).
+
+
+### Multiple Service CIDRs
+
+Support for clusters with multiple Service CIDRs moves to beta in v1.31 (disabled by default).
+
+There are multiple components in a Kubernetes cluster that consume IP addresses: Nodes, Pods and Services. 
+Nodes and Pods IP ranges can be dynamically changed because depend on the infrastructure or the network plugin respectively. 
+However, Services IP ranges are defined during the cluster creation as a hardcoded flag in the kube-apiserver. 
+IP exhaustion has been a problem for long lived or large clusters, as admins needed to expand, shrink or even replace entirely the assigned Service CIDR range. 
+These operations were never supported natively and were performed via complex and delicate maintenance operations, often causing downtime on their clusters. This new feature allows users and cluster admins to dynamically modify Service CIDR ranges with zero downtime.
+
+For more details about this feature please visit the
+[Virtual IPs and Service Proxies](/docs/reference/networking/virtual-ips/#ip-address-objects) documentation page.
+
+This work was done as part of [KEP #1880](https://github.com/kubernetes/enhancements/issues/1880) by [SIG Network](https://github.com/kubernetes/community/tree/master/sig-network).
+
+
+### Traffic distribution for Services
+
+Traffic distribution for Services moves to beta in v1.31 and is enabled by default. 
+
+After several iterations on finding the best user experience and traffic engineering capabilities for Services networking, the SIG Networking implemented the `trafficDistribution` field in the Service specification, that serves as a guideline for the underlying implementation to consider while making routing decisions.
+It supersedes the functionality formerly provided by the `service.kubernetes.io/topology-mode` annotation and its precursor `topologyKeys` field (which has been deprecated since Kubernetes 1.21).
```

Comment:
```suggestion
```

This one line in itself (without additional context about deprecations etc) may end up confusing people about: "Does service.kubernetes.io/topology-mode no longer work?" etc. Let's just include the first sentence? The link for the detailed docs (which you have already included) should help clarify all the other details.

Besides this, LGTM

Reply:
thanks for the suggestion, @gauravkghildiyal, i've updated the content to remove this sentence.